test/unit/tool/hand_tool_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:ui';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:paintroid/command/src/command_factory.dart';
+import 'package:paintroid/command/src/command_manager.dart';
+import 'package:paintroid/tool/src/hand_tool/hand_tool.dart';
+import 'package:paintroid/tool/src/tool_types.dart';
+
+import 'hand_tool_test.mocks.dart';
```

Comment:
please always use the full import with package:, not the relative path

Reply:
same as above.